Document air_quality triggers & conditions (#44955)
home-assistant/home-assistant.github.io: source/_conditions/air_quality.is_ozone_value.markdown

---
title: "Ozone value"
condition: air_quality.is_ozone_value
domain: air_quality
description: "Tests the ozone level of one or more entities."
related_conditions:
  - air_quality.is_no2_value
  - air_quality.is_voc_value
---

> The **Ozone value** condition passes when an ozone (O3) sensor's reading meets a specific level. Ground-level ozone is a key component of smog and tends to spike on hot, sunny afternoons. This condition helps your automation make smarter ventilation decisions, keeping the windows shut during a smog alert but allowing fresh air in once the ozone reading drops back to a comfortable level.

Reviewer: - Use "after" instead of "once" to describe sequence
- Simplify "drop back" -> "decrease"

```suggestion
The **Ozone value** condition passes when an ozone (O3) sensor's reading meets a specific level. Ground-level ozone is a key component of smog and tends to spike on hot, sunny afternoons. This condition helps your automation make smarter ventilation decisions, keeping the windows shut during a smog alert but allowing fresh air in after the ozone reading decreases to a comfortable level.
```

{% include integrations/labs_entity_triggers_note.md %}

{% include conditions/ui_header.md %}

> To use this condition in an automation:

Reviewer 2: ```suggestion
## Using this condition in an automation
```

    behavior: any
{% endexample %}

This passes when the outdoor ozone sensor reads at or above 100 µg/m³.

### Options in YAML

YAML sometimes provides additional options for more complex use cases that are not available through the UI.

{% options_yaml %}
threshold:
  description: >
    The ozone level the sensor has to meet or exceed for the condition to pass. Accepts a number or a reference to an `input_number`, `number`, or `sensor` entity.
  required: true
  type: any
behavior:
  description: >
    When multiple sensors are targeted, controls how results combine. Accepts `all` or `any`.
  required: true
  type: string
  default: any
{% endoptions_yaml %}

{% include conditions/targets.md %}

{% include conditions/behavior.md %}

## Good to know

- Sensors that are unavailable (`unavailable`) or have an unknown state (`unknown`) are skipped for **Any** and fail for **All**.
- Ozone levels tend to peak on hot, sunny afternoons. Pair this condition with time-based triggers to limit outdoor ventilation during those hours.

{% include conditions/try_it.md %}

{% include conditions/more_examples.md %}

### Automation: only open windows in the afternoon if ozone is low enough

> On hot summer days, ozone builds up through the afternoon and peaks around 3 PM. This automation triggers at that time and checks whether outdoor ozone is at or above 100 ug/m3. If the reading is still high, the windows stay shut and you get a notification explaining why. On breezy days with lower ozone, the automation does nothing and you enjoy fresh air as usual.

Reviewer: - use μg

```suggestion
On hot summer days, ozone builds up through the afternoon and peaks around 3 PM. This automation triggers at that time and checks whether outdoor ozone is at or above 100 μg/m3. If the reading is still high, the windows stay shut, and you get a notification explaining why. On breezy days with lower ozone, the automation does nothing, and you enjoy fresh air as usual.
```
